docs/plugins/virtual_environment_docs/1-virtual-environments.md

```diff
@@ -0,0 +1,68 @@
+# 1: Virtual environments  
+
+This guide explains the value of using virtual environments and how to create and remove them. 
+
+## Other topics in this series:  
```

Comment:
So, we should delete the statement that says, "The next topic in this series is..."? I moved the descriptions.  We have numbered them just because that is the order they were presented in.  You're right, they don't necessarily build on each other.  Should we remove the numbers too?

The last section is now: 

## Other topics in this series:  

* 2: [Deploying your plugin](./2-deploying-your-plugin.md)  
* 3: [Version management](./3-version-management.md)     
* 4: [Developer tools](./4-developer-tools.md)   
* 5: [Survey](./5-survey.md)

Reply:
@Nadalyn-CZI yeah agreed, I think we can get rid of the numbers here.